CONTRIBUTING.md

```diff
@@ -346,7 +346,7 @@ logo: "GTN"
 <!-- add a resized image in percentage: 10%, 25%, 50%, or 75% -->
 .image-25[![](../images/ecker_2012.jpg)]
 
-<small>[*Ecker et al, Nature, 2012*](http://www.nature.com/nature/journal/v489/n7414/full/489052a.html)</small>
+<small>[*Ecker et al, Nature, 2012*](https://www.ncbi.nlm.nih.gov/pubmed/22955614)</small>
```

Comment:
for me that redirects to the nature URL and is still giving me 401 in the travis check